docs/reference/snapshot-restore/repository-s3.asciidoc

```diff
@@ -262,7 +262,8 @@ multiple deployments may share the same bucket.
 `chunk_size`::
 
     (<<byte-units,byte value>>) Big files can be broken down into chunks during snapshotting if needed.
-    Specify the chunk size as a value and unit, for example:
+    When large file split into chunks, the chunk size will be defined by smallest of `chunk_size`
+    or `buffer_size * max_multipart_parts`. Specify the chunk size as a value and unit, for example:
     `1TB`, `1GB`, `10MB`. Defaults to the maximum size of a blob in the S3 which is `5TB`.
```

Comment:
I never really liked the wording here to start with :) How about something like this? 

```
    (<<byte-units,byte value>>) The maximum size of object that {es} will write to the repository when creating a snapshot. Files which are
    larger than `chunk_size` will be chunked into several smaller objects. {es} may also split a file across multiple objects to satisfy
    other constraints such as the `max_multipart_parts` limit. Defaults to `5TB` which is the
    https://docs.aws.amazon.com/AmazonS3/latest/userguide/qfacts.html[maximum size of an object in AWS S3].
```

Reply:
+ on doc changes, thank you